client/lib/abtest/active-tests.js

```diff
@@ -59,4 +59,13 @@ export default {
 		defaultVariation: 'control',
 		allowExistingUsers: false,
 	},
+	jetpackPopularPlans: {
+		datestamp: '20210304',
```

Comment:
We might need to update the date before launching.

Reply:
Yeah, let's go ahead and change this to Monday:

```suggestion
		datestamp: '20210308',
```